pandas/tests/io/json/test_normalize.py

```diff
@@ -370,10 +415,6 @@ def test_missing_meta(self, missing_metadata):
              'zip': 37643,
              'name': np.nan}
         ]
-        ex_data = [
```

Comment:
why did this change?

Reply:
It had both a `dict` and `list` with the same var.

Reply:
Yea this is not optimally written as is but I think removing the dict would be better than removing this list, since this is actually what gets used now anyway